lib/services/libvirtd.pm

```diff
@@ -0,0 +1,132 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Package for libvirtd service check tests for migration
+#
+# Maintainer: Wegao <wegao@suse.com>
+
+package services::libvirtd;
+use base 'opensusebasetest';
+use testapi;
+use utils;
+use strict;
+use warnings;
+
+#default guest vm params
+our %guest_params = (
+    'name'        => 'nested_vm_test',
+    'ram' => '',
+    'memory' => '512',
+    'vcpus'        => '',
+    'os-type'  => '',
+    'disk'  => 'none',
+    'network'  => '',
+    'graphics'  => 'vnc',
+    'boot'  => 'cdrom',
+);
+
+sub remove_repo {
+    my $repo_max_num = script_output("zypper lr | tail -1 | cut -d' ' -f1");
+    zypper_call("lr -u");
+    for (my $i = 1; $i < $repo_max_num + 1; $i = $i +1) {
+        zypper_call("rr 1");
+    }
+    script_run("zypper lr -u");
+}
+sub install_service {
+    if (get_var("DISTRI") eq "opensuse") {
+        remove_repo();
+        my $version = script_output("grep VERSION= /etc/os-release | cut -d'=' -f2 | cut -d' ' -f1 | sed 's/\"//g'");
```

Comment:
Wei, not a big deal, but you can take a look at "lsb_release -a" cmd
suse@localhost:~> lsb_release -a
LSB Version:	n/a
Distributor ID:	openSUSE
Description:	openSUSE Leap 15.2
Release:	15.2
Codename:	n/a